tests/pytorch/test_sanity.py

```diff
@@ -560,6 +545,8 @@ def test_sanity_grouped_linear(
             pytest.skip(reason_for_no_fp8)
         if fp8_recipe.mxfp8():
             pytest.skip("Grouped linear does not support MXFP8")
+        if fp8_recipe.float8_current_scaling():
```

Comment:
PR #1525 adds this support (just noting this since that one will only happen in 2.3 and this PR is for 2.2).